<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>411</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Create Bar </string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>281</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>ALS Bar UUID</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_als_bar_uuid"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_mf_bar_uuid"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_barName">
        <property name="text">
         <string>BARXXXXXX</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>MF Bar UUID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Bar Name</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton_create_bar_crucible">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>90</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Add to Crucible</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_create_bar_als">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>130</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Add to ALS DB</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_new_bar_name">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>50</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Get Next Bar #</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_upload_bar">
    <property name="geometry">
     <rect>
      <x>471</x>
      <y>510</y>
      <width>221</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload Bar Layout to Database</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_print_barcode">
    <property name="geometry">
     <rect>
      <x>701</x>
      <y>150</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Print Barcodes</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>411</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Thin Film Selection</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_tray_uuid">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>Scan Tray ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_tray_name">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TRAYXXXXX</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_add_to_bar">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>90</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Add One to Bar</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_select_barpos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Thin Film</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Bar Pos.</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_select_thinfilm">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>191</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_add_all">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>70</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Add All</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>421</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Crucible Login</string>
    </property>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>401</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0" colspan="2">
       <widget class="QLineEdit" name="username_lineEdit">
        <property name="text">
         <string>user name</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QComboBox" name="project_comboBox"/>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="session_lineEdit">
        <property name="text">
         <string>session name</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="tags_lineEdit">
        <property name="text">
         <string>tags,separated,by,commas</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLineEdit" name="comments_lineEdit">
        <property name="text">
         <string>comments</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QPushButton" name="logout_button">
        <property name="text">
         <string>Logout</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLineEdit" name="email_lineEdit">
        <property name="text">
         <string>user-email@lbl.gov</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_clear_bar">
    <property name="geometry">
     <rect>
      <x>701</x>
      <y>120</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Bar Layout</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>90</y>
      <width>221</width>
      <height>410</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Thin Film Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>mm </string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_tf_1"/>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_mm_1"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_tf_2"/>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_mm_2"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_tf_3"/>
     </item>
     <item row="3" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_mm_3"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_tf_4"/>
     </item>
     <item row="4" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_mm_4"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_tf_5"/>
     </item>
     <item row="5" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_mm_5"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEdit_tf_6"/>
     </item>
     <item row="6" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_mm_6"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="lineEdit_tf_7"/>
     </item>
     <item row="7" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_mm_7"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="lineEdit_tf_8"/>
     </item>
     <item row="8" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_mm_8"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="lineEdit_tf_9"/>
     </item>
     <item row="9" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_mm_9"/>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="lineEdit_tf_10"/>
     </item>
     <item row="10" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_mm_10"/>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>11</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="lineEdit_tf_11"/>
     </item>
     <item row="11" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_mm_11"/>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>12</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLineEdit" name="lineEdit_tf_12"/>
     </item>
     <item row="12" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_mm_12"/>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>13</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QLineEdit" name="lineEdit_tf_13"/>
     </item>
     <item row="13" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_mm_13"/>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>14</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QLineEdit" name="lineEdit_tf_14"/>
     </item>
     <item row="14" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_mm_14"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Incidence Angle:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>Offset from left (mm):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_offset"/>
     </item>
     <item row="0" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_angle"/>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_wafer_width"/>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>Wafer width (mm):</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>813</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGIWAXS_BAR_CREATOR">
    <property name="title">
     <string>GIWAXS BAR CREATOR</string>
    </property>
   </widget>
   <addaction name="menuGIWAXS_BAR_CREATOR"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
